website/docs/sys-mgmt/background-tasks.md

```diff
@@ -0,0 +1,76 @@
+- **Error**: the task failed to process. Either the task threw an exception, or reported an other error. Look at the task logs for further information. See [Failed tasks](#failed-tasks) for more details.
+- **Waiting to run**: the task has been queued for running, but no worker picked it up yet, either because none are available, they are already busy, or because it's just been queued.
+- **Running**: the task is currently running.
+
+## Manage background tasks
+
+### View system tasks
+
+Tasks and schedules can be viewed and managed from the admin interface.
+
+By default, tasks are shown _as close as possible_ to their relevant objects. For instance, LDAP source synchronization tasks and schedules are shown on the LDAP source detail page.
+
+When a task or a schedule cannot be associated to an object (for example housekeeping tasks), it is referred to as "standalone" and will be displayed under **Dashboards** > **System Tasks**. Note that tasks created from a schedule are associated to that schedule and are not considered standalone. Both schedule and task items can be expanded to view additional detail about them.
+
+If you cannot find the object to which a task or schedule is attached to, untick "Show only standalone tasks/schedules" on the **System Tasks** page to show all tasks and schedules, including the ones that are attached to objects.
+
+By default, successful tasks are hidden to minimize the number of shown items. Untick "Exclude successful tasks" to show them.
+
+### Schedule configuration
+
+When schedules are created, they are assigned a default interval. To change that interval, edit the schedule and update it. The format is a crontab.
+
+:::warning
+Some tasks are required to run at regular intervals. We only recommend editing synchronization schedules intervals for tuning reasons.
+:::
+
+Schedules can also be paused to prevent new tasks to be created from them. They can still be triggered manually while paused. When un-pausing a schedule, it will be triggered immediately.
+
+### Failed tasks
+
+When a task fails, i.e. when the code throws an exception, the task will be retried up to the value configured in [`AUTHENTIK_WORKER__TASK_MAX_RETRIES`](../install-config/configuration/configuration.mdx#authentik_worker__task_max_retries). Tasks that self-reported an error or a warning will not be retried.
+
+Failed tasks will be displayed as any other tasks. Each task can be expanded to show its logs. The logs are split into two parts: "Current execution logs" for the current execution, and "Previous execution logs" for logs from previous execution that happened before a retry was initiated. The information contained in the logs will point to a transient error (a network connection failed for example), a mis-configuration (wrong password set in the LDAP source for example) or a bug in authentik.
+
+#### Restarting tasks
+
+To restart a task, click the retry arrow next to the task. It will be queued again and picked up by a worker.
```

Comment:
```suggestion
To restart a task, click the arrow for the task to expand the details for that task and then click the Retry icon (icon.svg) next to the task. It will be queued again and picked up by a worker.
```

Reply:
Well, the retry icon doesn't need the task to be expanded. And it has nothing to do with schedules.

Reply:
Ah, so they are on the Tasks tab... not the schedules tab.